src/main/webapp/WEB-INF/views/css/header.css

```diff
@@ -0,0 +1,18 @@
+.appLogout {
```

Comment:
```suggestion
.app-logout {
```
use the same style for all classes